{{cookiecutter.project_slug}}/frontend/package.json

```diff
@@ -18,6 +18,7 @@
 		"@mui/icons-material": "^5.11.11",
 		"@mui/material": "^5.11.11",
 		"formik": "^2.2.9",
+		"rollup": "^4.34.9",
```

Comment:
minor thing: have you tried without rollup here and it doesn't work? if we can remove it as a dependency that's better